src/collections/Footer/config.ts

```diff
@@ -0,0 +1,160 @@
+import { accessByTenant } from '@/access/byTenant'
+import { filterByTenant } from '@/access/filterByTenant'
+import { contentHashField } from '@/fields/contentHashField'
+import { tenantField } from '@/fields/tenantField'
+import { generatePreviewPath } from '@/utilities/generatePreviewPath'
+import type { CollectionConfig, TextFieldValidation } from 'payload'
+
+const validateHashtag: TextFieldValidation = (value: string | null | undefined): string | true => {
+  if (value === null || value === undefined) return true
+  else
+    return (
+      value?.match(/^#[A-Za-z0-9_](?:[A-Za-z0-9_]|(?:\.(?!\.))){0,28}[A-Za-z0-9_]$/)?.length ===
+        1 || `${value} is not a valid hashtag`
+    )
+}
+
+const validateTelephone: TextFieldValidation = (
+  value: string | null | undefined,
+): string | true => {
+  console.log(value?.match(/^(\+1\s?|1\s?)?(\(\d{3}\)|\d{3})[\s.-]?\d{3}[\s.-]?\d{4}$/))
+  if (value === null || value === undefined) return true
+  else
+    return (
+      value?.match(/^(\+1\s?|1\s?)?(\(\d{3}\)|\d{3})[\s.-]?\d{3}[\s.-]?\d{4}$/g)?.length === 1 ||
+      `${value} is not a valid hashtag`
+    )
+}
+
+export const Footer: CollectionConfig = {
+  slug: 'footer',
+  access: accessByTenant('footer'),
+  admin: {
+    // the GlobalViewRedirect will never allow a user to visit the list view of this collection but including this list filter as a precaution
+    baseListFilter: filterByTenant,
+    group: 'Globals',
+    livePreview: {
+      url: async ({ data, req, payload }) => {
+        let tenant = data.tenant
+
+        if (typeof tenant === 'number') {
+          tenant = await payload.findByID({
+            collection: 'tenants',
+            id: tenant,
+            depth: 2,
+          })
+        }
+
+        const path = generatePreviewPath({
+          slug: '',
+          collection: 'pages',
+          tenant,
+          req,
+        })
+
+        return path
+      },
+    },
+  },
```

Comment:
Do we need the live preview? It just shows the home page. I think we can live without it.

Reply:
If it's hard to maintain, we can live without. If it's easy, I say why not! I think folks will appreciate having it everywhere.